resources/python/make_docker.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
+
+set -eu
+
+GIT_HASH="$(git rev-parse HEAD)"
+
+DOCKER_IMAGE_NAME="${1:-opinicus_${GIT_HASH}}"
+
+DOCKER_FILE_CHANGES=$(git diff docker_env/Dockerfile)
+
+REQUIREMENT_FILE_CHANGES=$(git diff requirements.txt)
```

Comment:
CMakeLists is leading in the Cura repositories (since requirements.txt doesn't work for non-Python code).

Reply:
Although I do think that we should swap that; We could also just let the cmakelists call the requirements.txt

Reply:
I agree with @nallath here. We should make the pip installable dependencies leading and use that in our cmakelists.
That makes settings up a project like Cura in a virtual environment much easier. You only have to export the PATH, LD_PATH and PYTHONPATHS for SIP/Python projects like Savitar, Arcus, Uranium